index.js

```diff
@@ -3,71 +3,95 @@ function ocr(str) {
 }
 
 function scanDigit(str) {
-  const zero =  " _ " + 
-                "| |" + 
-                "|_|"
+  let digitsNonWhiteSpace = [
+    {'digit': 0, 'nonWhites': 6, 'uniques': [
+      {'index': 4, 'char': ' '}, 
+      {'index': 5, 'char': '|'}, 
+      {'index': 6, 'char': '|'}]},
+    {'digit': 1, 'nonWhites': 2, 'uniques': []},
```

Comment:
Considering that the digit will be identified by the amount of `non whitespaces`, the idea behind this is to have a list of unique chars for similar digits.